tests/integrations/utils.py

```diff
@@ -154,6 +155,22 @@ def get_span_from_agent(weblog_request):
         raise ValueError(f"Span is not found for {weblog_request.request.url}")
 
 
+# set AWS credentials for runner
+os.environ["AWS_ACCESS_KEY_ID"] = os.environ.get("SYSTEM_TESTS_AWS_ACCESS_KEY_ID", "")
```

Comment:
Is there a way to pass those value to the sdk directly ? If it's possible to not hack to much `os.environ`, it's better.

Reply:
Changed this code, the code now just instantiates an AWS client session, and uses the appropriate variables without modifying the user's env.